wfl/descriptors/calc.py

```diff
@@ -76,9 +76,9 @@ def calc(inputs, outputs, descs, key, local=False, normalize=True, composition_w
 
```

Comment:
Is it worth renaming `calc` to something more descriptive? I am thinking of using it in code - I would be inclined to rename it when importing (like in `devtools/scripts/pretty_pca.py`) so it doesn't clash with other `calc` methods (if there are any) and the code using it is more readable.

Reply:
Can you do `from wfl import descriptors` and then use it as `descriptors.calc()`?  Anyway, let's wait until @gabor1 reminds me why we decided to move it into a subdirectory.

Reply:
Yes, that would work perfectly fine.

Reply:
It's now `descriptors.quippy.calc`. If that's OK, do you want to resolve this conversation?